### PR TITLE
Add activity and idea api

### PR DESCRIPTION
**Pull request by pk-nb**
@brandonc @ianisborn 

Adds docs for idea and comments. In conjunction with [napa PR](https://github.com/kapost/napa/pull/930) to expose a couple more routes.

Also I added a "Remove Field" button to the array fields to make testing that a little more sane.

idea

### CODE REVIEW

**Comment by brandonc** [comment]
Can you add the operation parameter to idea update method?

Possibile values are: reject, submit, approve, archive, unarchive

File: public/data/kapost_v1.json [hunk 1]
```diff
@@ -1,6 +1,305 @@
...
+            },
+            {
+              "Name":"per_page",
+              "Required":"N",
+              "Default":"25",
+              "Type":"integer",
+              "Description":"The number of entries to retrieve per page."
+            },
+            {
+              "Name":"user_id",
```
Comment (brandonc) [inline public/data/kapost_v1.json]: Even if this user_id param is real I think we should not expose it. It doesn't make sense to me.

Comment (pk-nb) [inline public/data/kapost_v1.json]: Sounds good. It was on content too, should I take it off?

File: public/data/kapost_v1.json [hunk 2]
```diff
@@ -1,6 +1,305 @@
...
+            },
+            {
+              "Name":"categories",
+              "Required":"N",
+              "Default":"",
+              "Type":"array",
+              "Description":"Filter ideas using the specified category name(s)"
+            },
+            {
+              "Name":"include_empty_categories",
```
Comment (brandonc) [inline public/data/kapost_v1.json]: Let's not expose this one either

**Pull request merged**